Cargo.lock

```diff
@@ -749,7 +749,7 @@ dependencies = [
  "semver 0.11.0",
  "semver-parser 0.10.2",
  "serde",
- "serde_json",
+ "serde_json 1.0.64",
```

Comment:
Why do we need exactly this version?

Reply:
It automatically set on building the app

Reply:
Can we avoid this?

Reply:
I haven't found any clear explanation why it happens. If the build/tests runs with `"serde_json"` and`--locked` flag it emit an [error](https://github.com/Cerebellum-Network/pos-network-node/actions/runs/3111083599/jobs/5042971487).
<img width="906" alt="image" src="https://user-images.githubusercontent.com/21261739/191919512-23913ac9-1ba7-4924-844f-3f2b65241916.png">
Also, there is no Cargo.toml file with version 1.0.64 of serde_json

Reply:
Please, create a separate task to fix this.

Reply:
[Created](https://cerenetwork.atlassian.net/browse/CBI-2246)